### ISSUE TITLE
Expansion simple elevator

### ISSUE THREAD

**Issue by Pvmdark** [SirBlobman/CombatLogX#239]
**Version Information:**
[01:07:01 INFO]: Server Version: git-Paper-428 (MC: 1.16.5)
[01:07:01 INFO]: Bukkit Version: 1.16.5-R0.1-SNAPSHOT
[01:07:01 INFO]: Minecraft Version: 1.16.5
[01:07:01 INFO]: NMS Version: 1_16_R3
[01:07:01 INFO]: 
[01:07:01 INFO]: CombatLogX by SirBlobman
[01:07:01 INFO]: Installed Version: v10.4.2.1
[01:07:01 INFO]: Latest Version: v10.4.2.1
[01:07:01 INFO]: 
[01:07:01 INFO]: Getting expansion versions...
[01:07:01 INFO]: 
[01:07:01 INFO]:   Cheat Prevention v15.3
[01:07:01 INFO]:   CrackShot Compatibility v15.0

**Which plugin needs a compatibility expansion?**
https://www.spigotmc.org/resources/simple-elevators-1-8-1-17.44462/

**Why do you need us to create an expansion?**
The developer of the plugin can't implement a combat system. 
Basically, I am making GTA in Minecraft and I want elevators on my server. But whenever the players use the elevator they shouldn't be able to use it when they are in combat. That is why I want this expansion.

**Extra Information:**
If I shoot with an RPG, the player gets knock backed so the system thinks the player jumps, is there a way to fix this?

**Comment by SirBlobman** [comment]
Thanks for the request, I'll check if there is an API for when players attempt to get on an elevator so that it can be prevented. I'll add another comment once I find out more information.

As far as RPG knockback, I'm pretty sure that should be an option in CrackShot. I haven't really used this plugin.

**Comment by SirBlobman** [comment]
Hello again, sadly the SimpleElevators plugin doesn't have a way to prevent a player from getting on an elevator. There are no custom events triggered, like CrackShot with their WeaponDamageEntityEvent. I'll close this for now, but if anything changes don't be afraid to reopen it.
